samples/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/gorilla/mux v1.7.2
 	go.opencensus.io v0.22.0
 	gocloud.dev v0.15.0
+	gocloud.dev/docstore/mongodocstore v0.0.0-00010101000000-000000000000
```

Comment:
This version looks weird, is it because it is a complete new module and gomod cannot find any revision?

Reply:
I guess?